src/gamepad/linux/linux-raw-gamepad.vala

```diff
@@ -0,0 +1,188 @@
+// This file is part of GNOME Games. License: GPLv3
+
+const int GUID_LENGTH = 8;
+
+private class LibGamepad.LinuxRawGamepad : Object, RawGamepad {
+	private int fd;
+	private uint? event_source_id;
+	private Libevdev.Evdev device;
+
+	private uint8 key_map[Linux.Input.KEY_MAX];
+	private uint8 abs_map[Linux.Input.ABS_MAX];
+	private Linux.Input.AbsInfo abs_info[Linux.Input.ABS_MAX];
+	private string identifier;
+	public string name { get { return device.name; } }
```

Comment:
Names are not used for _now_ so we should put them aside and add them later when we will need them.